.github/workflows/build-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      - 'feature/**'
+jobs:
+  run-build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Compose
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install docker-compose
+
+      - name: Build Docker Compose
+        run: |
+          if ! docker-compose build; then
```

Comment:
how long does this take to run usually out of curiosity?

Reply:
like 2 mins ish